src/components/flash/__spec__.js

```diff
@@ -215,12 +215,14 @@ describe('Flash', () => {
 
     describe('if the text does contains ::more::', () => {
       it('returns the text and creates an alert', () => {
-        spyOn(defaultInstance, 'toggleDialog');
+        const toggleDialogSpy = jasmine.createSpy();
```

Comment:
Can you explain why these four lines within this `it` have changed, extra values and calls in setp and expectations shouldn't be occurring because of linting changes